### PR TITLE
DT-571: Add prison-estate-prod new service

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: estate-dev
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/00-namespace.yaml]: Can you please change it to prison-estate-dev, same as your folder name.
OR if you like to keep this name, can you please change your folder name.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/01-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: estate-dev-admin
+  namespace: estate-dev
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/01-rbac.yaml]: Same here, please match with the folder name

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-preprod/00-namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: estate-preprod
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-preprod/00-namespace.yaml]: Can you please change it to prison-estate-preprod, same as your folder name.
OR if you like to keep this name, can you please change your folder name.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/00-namespace.yaml (hunk 1 above)
Comment (petergphillips) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/00-namespace.yaml]: Apologies - over zealous search and replace went on.  Dev prison estates will be on the domain https://estate-dev.prison.service.justice.gov.uk so my replace for the domain affected all the namespaces by accident.  Now fixed :crossed_fingers:

**Pull request merged**